tests/acceptance/features/apiSharingNgDriveInvitation/driveShareInvitations.feature

```diff
@@ -541,6 +533,7 @@ Feature: Send a drive invitations
       | Space Viewer     | role not applicable to this resource |
       | Space Editor     | role not applicable to this resource |
       | Manager          | role not applicable to this resource |
+      | Secure Viewer    | role not applicable to this resource |
```

Comment:
Reason
- Secure Viewer is additional role (I think not implemented upto now since long time).
- another reason is same as below 

```suggestion
```

Reply:
let's only add tests for the default available roles for Personal and Shares space scenario